test/test_unistd.rs

```diff
@@ -90,7 +91,7 @@ mod linux_android {
 
     #[test]
     fn test_gettid() {
-        let tid = gettid();
+        let tid: ::libc::pid_t = gettid().into();
```

Comment:
I'm pretty certain this is always going to be > 0 (related to `getpid` test above).